app/assets/images/icons/facebook.svg:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]

```

Comment:
@adrianMRJ  Sorry, what do you need these files for?

Reply:
They are auto generated, we delete them but it reapears